app/api/crud.py

```diff
@@ -85,6 +85,7 @@ async def get(
                 os.environ.get(util.GRAPH_USERNAME.name),
                 os.environ.get(util.GRAPH_PASSWORD.name),
             ),
+            timeout=10.0,
```

Comment:
1. I think this means you can also close https://github.com/neurobagel/api/issues/140 with this
2. I also think that 10 seconds is not enough given the query times we have already seen - until we fix the underlying issue of unoptimized and slow queries, let's not set this value too low. How about a) a TODO comment to revisit the value when performance improves, b) 30 seconds?